### PR TITLE
Message in reaction roles, Stats fixing

### PR DESCRIPTION
**Pull request by vinc3nz**
Not yet done. Will doing this in the next days maybe.

**This code will mostly break the whole bot. This time it is really necessary to review the code**

### CODE REVIEW

**Comment by vinc3nz** [comment]
Dan being funny again.

File: bot/discord/events/reactionAdd.js [hunk 1]
```diff
@@ -35,7 +35,12 @@ module.exports = async(client, r, member) => {
         let role = member.guild.roles.cache.get(found.role);
         if (member.roles.cache.get(found.role) == null) {
             await member.roles.add(role.id);
-            member.user.send("Gave you the role: `" + role.name + "`!");
+            client.channels.cache.get("898041838701781013").send(`<@${member.id}>\nGave you the *${role.name}* role! To remove it click the reaction again.`)
+            .then(msg => {
+               msg.delete(10000)
+            })
+            .catch(e => console.log(e));
+            //member.user.send("Gave you the role: `" + role.name + "`!");
```
Comment (itzJOHv) [inline bot/discord/events/reactionAdd.js:43]: why did you need to make it like this?

File: bot/discord/events/reactionRemove.js [hunk 2]
```diff
@@ -37,7 +37,12 @@ module.exports = async(client, r, member) => {
     let role = member.guild.roles.cache.get(found.role);
     if (member.roles.cache.get(found.role) != null) {
         await member.roles.remove(role);
-        member.user.send("Removed the role: `" + role.name + "`!");
+        client.channels.cache.get("898041838701781013").send(`<@${member.id}>\nRemoved the *${role.name}* role from you! To add it again click the reaction again.`)
+        .then(msg => {
+           msg.delete(10000)
+        })
+        .catch(e => console.log(e));
+        //member.user.send("Removed the role: `" + role.name + "`!");
```
Comment (itzJOHv) [inline bot/discord/events/reactionRemove.js:45]: same to this

**Comment by omxpro** [comment]
Can you add confirm button to delete cmd

**Comment by Sliden101** [comment]
image
deprecated

**Comment by vinc3nz** [comment]
image

Just adding a few more infos and the stats work again

**Comment by vinc3nz** [comment]
@danielpmc
I think everything is done. But one question: How big are the disks? Are it about 300GB/server or are it Mb? Because I now multiply it by 1000 to show the size in Gb

**Comment by danielpmc** [comment]
Its different per node. 

> @danielpmc I think everything is done. But one question: How big are the disks? Are it about 300GB/server or are it Mb? Because I now multiply it by 1000 to show the size in Gb

**Comment by vinc3nz** [comment]
But for example Node 6 has 300GB and not 300MB?

**Comment by vinc3nz** [comment]
I have removed the buttons I added.
The stats command should work fine. I will edit it over the time to provide single node infos etc.
In reaction roles there is a medsage semd saying you got the role. The message sill be deleted after a few seconds

**Comment by DEV-DIBSTER** [comment]
Will be closing due to being outdated as well as conflicts.
